[2024-10-15T19:22:41.120+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-15T19:22:41.253+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-10-15T19:22:41.280+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-02-08T00:00:00+00:00 [queued]>
[2024-10-15T19:22:41.285+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-15T19:22:41.360+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): hello_world_task2> on 2024-02-08 00:00:00+00:00
[2024-10-15T19:22:41.376+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1399) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-15T19:22:41.382+0000] {standard_task_runner.py:63} INFO - Started process 1441 to run task
[2024-10-15T19:22:41.392+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'hello_world_task2', 'scheduled__2024-02-08T00:00:00+00:00', '--job-id', '274', '--raw', '--subdir', 'DAGS_FOLDER/example_hello_world.py', '--cfg-path', '/tmp/tmp1kxc77w8']
[2024-10-15T19:22:41.402+0000] {standard_task_runner.py:91} INFO - Job 274: Subtask hello_world_task2
[2024-10-15T19:22:41.572+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_world_dag.hello_world_task2 scheduled__2024-02-08T00:00:00+00:00 [running]> on host f9b6cc3118f7
[2024-10-15T19:22:42.043+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='hello_world_task2' AIRFLOW_CTX_EXECUTION_DATE='2024-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-08T00:00:00+00:00'
[2024-10-15T19:22:42.066+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-15T19:22:42.070+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-15T19:22:42.074+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'Hello World! Now I am triggered after the first task!'"]
[2024-10-15T19:22:42.153+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T19:22:42.167+0000] {subprocess.py:93} INFO - Hello World! Now I am triggered after the first task!
[2024-10-15T19:22:42.169+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T19:22:42.171+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-15T19:22:42.297+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=hello_world_task2, run_id=scheduled__2024-02-08T00:00:00+00:00, execution_date=20240208T000000, start_date=20241015T192241, end_date=20241015T192242
[2024-10-15T19:22:42.407+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-15T19:22:42.566+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-15T19:22:42.589+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
